lang/en.json

```diff
@@ -754,6 +754,8 @@
         "description": "Description",
         "price_with_tax": "(Inc. {tax_label})",
         "price_without_tax": "(Ex. {tax_label})",
+        "price_range_from": "from",
+        "price_range_to": "to",
```

Comment:
Could we use existing from, to fields?

Reply:
fair
took shop_by_price_range_aria from en.json you created earlier